docs/swagger.json

```diff
@@ -0,0 +1,651 @@
+{
+	"swagger": "2.0",
+	"info": {},
+	"basePath": "/corellia/api",
+	"tags": [
+		{
+			"name": "Corellia"
+		}
+	],
+	"paths": {
+		"/version": {
+			"get": {
+				"tags": [
+					"Corellia"
+				],
+				"summary": "returns version",
+				"description": "",
+				"operationId": "getVersion",
+				"produces": [
+					"text/plain"
+				],
+				"parameters": [],
+				"responses": {
+					"200": {
+						"description": "successful operation",
+						"schema": {
+							"type": "string"
+						}
+					}
+				}
+			}
+		},
+		"/contracts": {
+			"post": {
+				"tags": [
+					"Corellia"
+				],
+				"summary": "Saves a contract",
+				"description": "",
+				"operationId": "saveContract",
+				"produces": [
+					"application/json"
+				],
+				"parameters": [
+					{
+						"in": "body",
+						"name": "body",
+						"required": false,
+						"schema": {
+							"$ref": "#/definitions/Contract"
+						}
+					}
+				],
+				"responses": {
+					"200": {
+						"description": "successful operation",
+						"schema": {
+							"type": "string"
+						}
+					}
+				}
+			}
+		},
+		"/documents": {
```

Comment:
Can we clarify here the order of operations, as it's unclear.

In previous discussions with Frank it was.

1.  Upload document, which returns a token
2. Make policy request, including token for document so you know what to copy in background.

Here, it looks like

1. Upload policy?
2. Upload documents with policy Id reference. 

Could you clarify?  I think at least we should be saying in policy what document as expected

Reply:
yes your first assumption is correct:

step 1. Upload document, which returns a token
step 2. Upload document ....
 ... when all documents are uploaded:
step n. Make policy request, including tokens for documents so you know what to copy in background.